.github/workflows/linux-builds.yml

```diff
@@ -71,6 +71,13 @@ jobs:
           '*/src/mock-log.h' \
           '/usr/*' \
           --output-file coverage.info
+
+        for file in $(ls src/glog/*.h.in); do
```

Comment:
nit: I don't think `ls` is needed here, `for f in *.h; do` should work just fine.